js/apps/keycloak-server/README.md

```diff
@@ -41,6 +41,16 @@ If you want to run with a local Quarkus distribution of Keycloak for development
 pnpm start --local
 ```
 
+If you want to run with `IDELauncher`, make sure you set the following environment variables.
+
+```sh
+export KC_ACCOUNT_VITE_URL=http://localhost:5173
+export KC_ADMIN_VITE_URL=http://localhost:5174
+export KC_FEATURES=login2,account3,admin-fine-grained-authz,transient-users,oid4vc-vci
```

Comment:
Let's not add the `export` statement here, most people will configure these in their IDE and this makes for easier copy-pasting.
```suggestion
KC_ACCOUNT_VITE_URL=http://localhost:5173
KC_ADMIN_VITE_URL=http://localhost:5174
KC_FEATURES=login2,account3,admin-fine-grained-authz,transient-users,oid4vc-vci
```